Cornelica/scenario/rp01a.txt

```diff
@@ -68,73 +70,74 @@
 @se f=挿入音
 @bgf bg=ev01a
 
-[リリィ/rry0028]「んあっ！　あっあっ……んはぁあああ……あっふ……んふううっ」
+[Lily/rry0028]"Ah! Ah...ha ha...uhm..."
 
 @se f=愛液1 pitch=50
 
-[リリィ/rry0029]「ほら、挿入ってる！　これ……私の膣中にアルのおちんちん……挿入っちゃってるよ」
+[Lily/rry0029]"Look, I'm put it in! Here... I'm inserting it... your penis in my vagina..."
 
 @se f=愛液1 pitch=50
 
-[リリィ/rry0030]「んふうう……これ……おちんちん……大きい。あっあっ……私の膣中……広げられてくのが分かる。んくっ……はふんんっ」
+[Lily/rry0030]"Fuuu...this...penis.. is large. Ah... i can feel it spreading out...in my vagina. N...argh"
 
-[リリィ/rry0031]「届いてる。アルのおちんちんが奥まで……。子宮に当たってるのが分かる。あはぁあ……凄く熱い……。おまんこの膣中……火傷しちゃいそうなくらい」
+[Lily/rry0031]"It reaches. Al penis is deep down... You know you are hitting the uterus. Ah...It's really hot...My vagina... it's like it burns"
 
-[リリィ/rry0032]「はぁっはぁっはぁっ……どう？　気持ちいい？　私のおまんこ……感じる？」
+[Lily/rry0032]"Hahha... How is it? Does it feel good? My vagina...do you feel it? 
 
-[アル]「ぅあっ！　……ぁ……あああ！」
+[Al]"Ah! ...A..ahhah!
 
-[リリィ/rry0033]「返事をする余裕もないか。んふふ、おちんちんが溶けちゃいそうなくらい感じてるって感じだね。あふぅう……」
+[Lily/rry0033]"You can't even reply. Fufu♪, looks like you feel that the penis is about to melt...Afu ~uu.."
 
 @se f=愛液1 pitch=50
 
-[リリィ/rry0034]「本当に気持ち良さそう。膣中でおちんちんが震えてるのが分かる。ビクビクが伝わって来てるよ。あっは……んはぁあああ……」
+[Lily/rry0034]"It looks really pleasant. I can see that the penis is trembling in the vagina. [?] is coming down. Awwa…… n ha~aha……"
+;;「本当に気持ち良さそう。膣中でおちんちんが震えてるのが分かる。ビクビクが伝わって来てるよ。あっは……んはぁあああ……」
+[Lily/rry0035]"This... just feeling a penis...I...well...Haaaaaaaaaaaaaaa...nice... I feel so much"
 
-[リリィ/rry0035]「これ……おちんちんを感じてるだけで私も……んふう……はぁっはぁっ……いい……すごく感じちゃう」
-
-[リリィ/rry0036]「でも、まだ我慢。アルもまだ我慢だよ。もっと……もっともっと……気持ち良くなるためにね」
+[Lily/rry0036]"But I still have endurance. Al still endurance. More...more, more... to become more pleasant!"
```

Comment:
This line kinda sounds odd to me, but I'm not sure how to better write it.

Reply:
Welcome to my struggle e.e